### ISSUE TITLE
[Request] Add Ability to Rename Pokemon

### ISSUE THREAD

**Issue by inSaiyanxo** [nicoschmitt/necrobotvisualizer#44]
Hi, this visualizer is great! An additional option allowing us to rename Pokemon would be awesome too. This allows people to manually rename their Pokemon based on the IVs displayed.

**Comment by niicojs** [comment]
the bot is supposed to do that I think. you want to do it manually?

**Comment by inSaiyanxo** [comment]
Yep the bot manually does it for pokemon above the specified IV, but I would like to manually do it for specific Pokemon. It's not that important though, just a suggestion :)

**Comment by niicojs** [comment]
I'll add it to my todo list :)
